shared/video-sdk/authentication-workflow/project-implementation-uikit/react-native.mdx

```diff
@@ -0,0 +1,60 @@
+<PlatformWrapper platform="react-native">
+
+1.  **Add variables for your connection to the token server**
```

Comment:
Are you sure about this? If the React UI Kit works in the same way as all the others, all you need to do is add a variable with the server URL. Can you double-check please.

Reply:
These variables are just query params sent to the token server. So that's `role`, `expireTime` and the token server URL. I can hardcode these values into the URL as well, for now these are taken as variables and concatenated to the URL.

Reply:
![image](https://user-images.githubusercontent.com/42718091/188431113-5c4e0c64-e505-4d5d-90a2-4903568ce207.png)

Reply:
Resolved